packages/blocks/src/shape-block/shape-model.ts

```diff
@@ -0,0 +1,24 @@
+import { BaseBlockModel, IBaseBlockProps, Page } from '@blocksuite/store';
+import type { ShapeColor, ShapeType } from '../__internal__';
+
+export interface ShapeBlockProps extends IBaseBlockProps {
+  flavor: 'affine:shape';
+  color: ShapeColor;
+  type: ShapeType;
+
+  // Refs: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/d
+  d: string;
```

Comment:
Should we add the `d` and `color` fields to the `ShapeBlockModel`?

Reply:
Color field haven't been used but must be used in the future.